src/operator/activation.cc

```diff
@@ -65,16 +65,18 @@ Operator *ActivationProp::CreateOperatorEx(Context ctx, std::vector<TShape> *in_
 DMLC_REGISTER_PARAMETER(ActivationParam);
 
 MXNET_REGISTER_OP_PROPERTY(Activation, ActivationProp)
-.describe(R"code(Elementwise activation function.
-The activation operations are applied elementwisely to each array elements. The
-following types are supported:
+.describe(R"code(Element-wise activation function.
+The activation operations are applied element-wisely to each array elements.
```

Comment:
suggest changing "element-wisely to each array elements" to "element-wise to each array element"